Post: Intelligence Cycle

sroberts opened the pull request:
Finally continuing posts on key intelligence concepts by writing up the intelligence cycle.

krmaxwell commented:
A few thoughts as I read the article:

> In most cases companies using the word intelligence are trying to say something involves an intelligent concept. In many cases these are innovative and beneficial, but they aren't intelligence. 

- Two different (and equally valid) uses of the word _intelligence_. The real problem is when somebody uses the word without clearly meaning one definition or the other.
- Your aside on group naming could take a little more exposition. Naming is a real problem here, but it's a nuanced one because of both OPSEC and marketing pressures.
- CRITs link is broken.
- I believe that, in addition to the larger feedback phase from the policymakers / report audience, each phase transition should also incorporate a smaller feedback loop. Your processing will end up informing how you do collection, for example.

This is a fantastic post. Take a little extra time and make sure it's totally polished because this should end up being a reference for many people for a long time. Think about having it published someplace else, too.

sroberts commented:
- I added a section to address the first part, including a bit of a non-subtle dig at those who confuse those deliberately.
- I added a bit more exposition, but I honestly might do a full post just about that issue.
- The CRITs link is not broken, you can tell that by the :white_check_mark: above.
- I actually believe this 4th point is organic.
